components/connector-service/scripts/can-i-commit.sh

```diff
@@ -24,15 +24,15 @@ if [ $? != 0 ]
 fi
 
 ##
-# MAKE BUILD
+# MAKE VERIFY
```

Comment:
Same as in `application-registry`, not sure if we still need it.

Reply:
Done!